### ISSUE TITLE
Check for dark tag/comment/something in http/css to not invert

### ISSUE THREAD

**Issue by CobaltSpace** [darkreader/darkreader#1179]
So, the idea is that, websites have their own dark themes, and some are just dark by default.  So the idea is that a web developer could put in something saying that a section is dark, and then Dark Reader will just not do anything to it.  This way, instead of us having universal dark lists, we could just have it check for a dark tag/comment.

Also have the option to ignore dark tag/comment for when it is abused.

Basically, I am suggesting giving some control to web developers.

**Comment by spikespaz** [comment]
I was about to open my own issue asking for an exclusion list, and found this.
I like the idea, if, for example, my website (my name dot com) already has a dark theme.

In fact, it was inspired by Dark Reader's generated theme when the site used to be white.
I can imagine some reluctance because it belies the "automatic" part of this extension, however I believe the idea is sound because every site added would have to be peer-reviewed and approved in a pull request to this repository. That allows repo maintainers to decide if the dark theme of a particular website is complete enough to take precedence over DR.

**TL;DR**: It'd be neat if I could just add my domain to a list in this repository to be ignored by DR.

**Comment by Gusted** [comment]
#2855 
We added a meta tag for <meta name="darkreader" instance="UUID here"> and if it's found it wouldn't do anything.
